spec/spec.md

```diff
@@ -683,8 +688,10 @@ historical state for a given [[ref:DID]]. The following API can be used with spe
   - `seq` - **integer** - **OPTIONAL** - Sequence number of the DID to resolve
 - **Returns**:
   - `200` - Success.
-    - `did` - **object** - A JSON object representing the DID Document.
-    - `types` - **array** - An array of [type strings](#type-indexing) for the DID.
+    - `did` - **object** - **REQUIRED** - A JSON object representing the DID Document.
+    - `pkarr` - **string** - **REQUIRED** - An unpadded base64URL-encoded representation of the full [[ref:Pkarr]] payload, represented as 64 bytes sig,
+    - `types` - **array** - **OPTIONAL** - An array of [type integers](#type-indexing) for the DID.
+    8 bytes u64 big-endian seq, 0-1000 bytes of v concatenated; enabling independent verification.
```

Comment:
I think the `8 bytes u64 big-endian...` text should appear after the `- pkarr` line:

```suggestion
    - `did` - **object** - **REQUIRED** - A JSON object representing the DID Document.
    - `pkarr` - **string** - **REQUIRED** - An unpadded base64URL-encoded representation of the full [[ref:Pkarr]] payload, represented as 64 bytes sig,
    8 bytes u64 big-endian seq, 0-1000 bytes of v concatenated; enabling independent verification.
    - `types` - **array** - **OPTIONAL** - An array of [type integers](#type-indexing) for the DID.
```